README.MD

```diff
@@ -150,13 +150,24 @@ This will disable the log so will **NOT** see this.
     /Users/wswebcreation/multiple-cucumber-html-reporter/.tmp/index.html
 ========================================================================
 ```
+### `pageTitle`
+- **Type:** `string`
+- **Mandatory:** No
+
+You can change the report title on HTML head Tag
 
 ### `reportName`
 - **Type:** `string`
 - **Mandatory:** No
 
 You can change the report name to a name you want
 
+### `pageFooter`
+- **Type:** `string`
+- **Mandatory:** No
+
+You can customise Page Footer if required
```

Comment:
can you provide an example how it then should like like?

Reply:
Page Footer is where it has "Created by wswebcreation. Find me on ***ICONS***"
when i set 
pageFooter: '<span>my own footer</spn>' 
in report.generate({}) then it overwrite the existing Tag line